docs/installation/troubleshooting/bootstrap_installation_errors.rst

```diff
@@ -0,0 +1,46 @@
+Bootstrap Installation Errors
+=============================
+
+Bootstrap installation fails with no straightforward reason
+-----------------------------------------------------------
+
+If during a MetalK8s installation you encounter a failure and the console
+output does not provide enough information to identify the cause of
+the failure, re-run the installation with the verbose flag (``--verbose``).
+
+.. parsed-literal::
+
+   root@bootstrap $ /srv/scality/metalk8s-|version|/bootstrap.sh --verbose
+
+Errors after restarting the bootstrap node
+------------------------------------------
+
+If you reboot the bootstrap node and for some reason some containers
+(especially the salt-master container) refuse to start, perform the
+following checks:
+
+#. Ensure that the MetalK8s ISO is mounted properly.
```

Comment:
@gdemonet  is this the only reason a container might not restart?
The paragraph says "Perform the following checks", then "Check the ISO is mounted" --> if not do this.
Are there other checks?
@cathydossantospinto  if not, we can reword: "If some containers refuse to start after rebooting the boostrap node, ensure the MetalK8s ISO is properly mounted. <command>. If not, run the following command to check the status of the ISO file and remount it."

Reply:
I'm not sure this section is relevant anymore, TBH. A successful bootstrap installation should never lead into this state, because this `archives.mounted` formula is one of the first to run. Maybe @TeddyAndrieux (original author) has an opinion?

Overall, I think the "Troubleshooting" guide deserves a more in-depth review/rework, but for the topic of this PR (which is about restructuring content), we shouldn't spend too much time on it.